src/eval.py

```diff
@@ -503,7 +503,7 @@ def read_run_dir(run_dir):
                 all_runs[k].append(v)
 
     df = pd.DataFrame(all_runs).sort_values("run_name")
-    assert len(df) == len(df.run_name.unique())
+    #assert len(df) == len(df.run_name.unique())
```

Comment:
I think we should address what's causing this to fail vs commenting it out

Reply:
I've removed the assert, but it seems to be working? I'm not sure what run is failing, I've been running linear_mlp.yaml and it seems fine.